app/src/main/java/org/gdg/frisbee/android/arrow/SummitNotificationReceiver.java

```diff
@@ -12,7 +15,8 @@ public SummitNotificationReceiver() {
 
     @Override
     public void onReceive(Context context, Intent intent) {
-        Intent serviceIntent = new Intent(context, SummitNotificationService.class);
+        Intent serviceIntent = new Intent(context, TaggedEventSeriesNotificationService.class);
+        serviceIntent.putExtra(Const.EXTRA_TAGGED_EVENT, intent.getParcelableExtra(Const.EXTRA_TAGGED_EVENT));
```

Comment:
The below line will handle passing all the extras. We don't need to pass this explicitly.